README.md

```diff
@@ -19,24 +19,34 @@ Opni currently features log anomaly detection for Kubernetes.
   * Ability to setup & send alerts (slack/email/etc) based on Opni log insights
 
 ----
-#### Try it out!
+### Install Opni in your Kubernetes cluster:
+* Download the `opnictl` binary from the [latest release](https://github.com/rancher/opni/releases/tag/v0.1.2)
+* Install Opni using `opnictl`
+  ```
+  opnictl install
+  opnictl create demo
+  ```
+  * Will look at `~/.kube/config` for your cluster's kubectl configuration file
+  * Cluster Hardware requirements: 4 vCPUs, 16GB RAM
+    * *2 Nvidia GPUs* required if you want the AI to learn from your workloads (recommended)
```

Comment:
Maybe we should mention vGPUs? You could use a single gpu split into 2 with VCS.

Reply:
We haven't tested in this setting so not comfortable stating vGPUs will work for our training/inferencing requirements.